tests/integration/test_http_transport.py

```
@@ -0,0 +1,83 @@
+@pytest.mark.integration
+def test_http_server_rejects_missing_port():
+    """Test that server fails without port in HTTP mode."""
+    env = os.environ.copy()
+    env.update({
+        "MCP_TRANSPORT": "http",
+        "MCP_HOST": "127.0.0.1",
+    })
+    env.pop("MCP_PORT", None)  # Remove port
+
+    process = subprocess.Popen(
+        ["python", "-m", "mcp_zammad"],
```

Ruff (0.14.4)

Starting a process with a partial executable path

(S607)